crates/gitbutler-tauri/src/virtual_branches.rs

```diff
@@ -19,6 +19,7 @@ pub mod commands {
     use tauri::State;
     use tracing::instrument;
 
+    use crate::workspace::StackEntry;
```

Comment:
@krlvi this dependency isn't cool, but what can we do? I'd want to return the correct type so that we can use the response in the stack service.

Reply:
It's not a problem but you will likely need to do this earlier in the flow (see the other comment)